app/models/services/base-service.ts

```diff
@@ -189,7 +189,7 @@ export default abstract class BaseService<ServiceParamType> {
    * received.
    * @param _logger - the logger associated with the request
    */
-  protected abstract async _run(_logger: Logger): Promise<InvocationResult>;
+  protected abstract _run(_logger: Logger): Promise<InvocationResult>;
```

Comment:
I didn't remove it because of redundancy, I removed it because I was getting a compiler warning that said it was incorrect to label an abstract method as `async`.